apps/web/src/icons/index.tsx

```diff
@@ -0,0 +1,82 @@
+import AlertIcon from './AlertIcon';
```

Comment:
pls advise if there's a way to automate this/ import+export all the icons from the directory at once

Reply:
You could try this https://webpack.js.org/guides/dependency-management/#requirecontext but I haven't used it 🤷